lib/index.ts

```diff
@@ -319,10 +330,12 @@ export function main() {
     const ext = path.extname(filePath);
     const dir = outDir || path.dirname(filePath);
     const outPath = path.join(dir, path.basename(filePath, ext) + suffix + (options.format === "ts" ? ".ts" : ".js"));
-    if (verbose) {
-      console.log(`Compiling ${filePath} -> ${outPath}`);
+    if (checkUpdate(filePath, outPath)) {
```

Comment:
If the update isn't needed, let's log something when `verbose` is set (e.g. `Skipping ${filePath} because ${outPath} is newer`?)

And secondly, I think there should be a flag to make this behavior optional. I don't think all users want it (e.g. when `-ti` files are checked into a version-control system, they may not have a reliable timestamp on checkout, merging, etc.) Maybe the new behavior is a better default, but to avoid surprises to existing users, I'd rather add a flag to turn on this behavior. E.g. `-t -- Skip the build if the output file exists with a newer timestamp`.